_posts/2020-07-21-tech-stack.md

```diff
@@ -0,0 +1,368 @@
+---
+layout: post
+title: Tech Stack
+date: '2020-07-21'
+style: generic
+stacks:
```

Comment:
Good idea. Consider if this data should be moved out of this file so it's easier to find a change the stack independently of the rest of the article.